### DISCUSSION TITLE
Simultaneous keypress

### DISCUSSION THREAD

**Discussion by Mules75** [CommandPost/CommandPost#3377] [Technical Support]
I have a Midifighter Twister hooked up to a MacBook Pro M1 16". And I have been able to succesfully send midi to keystrokes and control various graphic applications. 
But I havent found how to setup that a press on a miditwister knob sends a simultaneous keypress of the two keyboard keys. Is this possible ? I was able to do this with an applescript but this creates a delay if you run this from another application.

Thanks!

Midifighter Twister - https://store.djtechtools.com/products/midi-fighter-twister?srsltid=AfmBOooVp4HIxF4Tg8gT3SH74tLJ2m2ob0aDK55d4WAvhY7Qw-u71nz3

**Comment by latenitefilms** [comment]
You can use Lua Snippets for this.

You can use [`hs.eventtap.keyStroke(modifiers, character[, delay, application])`](http://www.hammerspoon.org/docs/hs.eventtap.html#keyStroke) if you want to send a modifier key and a normal key.

You can use [`hs.eventtap.event.newKeyEvent([mods], key, isdown)`](http://www.hammerspoon.org/docs/hs.eventtap.event.html#newKeyEvent) if you want to hold down more than one "normal" key.

**Reply by Mules75** [reply]
Ok thanks!! Not familiar with Lua, but i'll do some research,
